README.md

```diff
@@ -384,6 +384,11 @@ Pathfinder has a monitoring API which can be enabled with the `--monitor-address
 
 `/ready` provides a way of checking whether the node's JSON-RPC API is ready to be queried. It returns a `503 Service Unavailable` status until all startup tasks complete, and then `200 OK` from then on.
 
+### Metrics
+
+`/metrics` provides a [Prometheus](https://prometheus.io/) metrics scrape endpoint. Currently only one type of metric is available:
+- `rpc_method_calls_total` counter, where the key `method` should be used to point to a particular RPC method to retrieve that method's total call count, for example: `rpc_method_calls_total{method="starknet_getStateUpdate"}`.
```

Comment:
I think `method` should be called a _label_, not a _key_: https://prometheus.io/docs/practices/naming/#labels

Reply:
This was actually a mental shortcut for a _label key_:
- `method` is the label _key_,
- `starknet_getStateUpdate` is the label _value_,
- `method="starknet_getStateUpdate"` is the actual _label_.

How about:
```suggestion
- `rpc_method_calls_total` counter, where the label key `method` should be used to point to a particular RPC method to retrieve that method's total call count, for example: `rpc_method_calls_total{method="starknet_getStateUpdate"}`.
```

Reply:
Thanks, I think it's much better.